app/main.js

```diff
@@ -39,6 +41,7 @@ let microbreakIdeas
 let breakIdeas
 let breakPlanner
 let appIcon = null
+let appIcon2 = null;
```

Comment:
This won't be needed anymore. 

Also, I am using [standardjs](http://standardjs.com/) but I guess your editor is making some auto-changes that go against standardjs. You can see errors with `npm run lint` but I think we can leave this as a final step.